src/main/java/mate/academy/exception/RegistrationException.java

```diff
@@ -0,0 +1,6 @@
+package mate.academy.exception;
+
+public class RegistrationException extends RuntimeException {
+    public RegistrationException(String s) {
+    }
```

Comment:
The constructor should call `super(s);` to pass the message to the superclass `RuntimeException`.